packages/deployment/test/test_lambda.js

```diff
@@ -103,20 +138,20 @@ test.serial(
       `${Lambda.messageAdapterZipFileHash}-${path.basename(t.context.lambda.remote)}`
     );
 
-    fs.writeFileSync(existingLambdaLocal, 'hello');
-    t.is(fs.statSync(existingLambdaLocal).size, 5);
+    fs.copySync(`${t.context.fixturedir}/zipfile-fixture.zip`, existingLambdaLocal);
+    t.is(fs.statSync(existingLambdaLocal).size, 180);
```

Comment:
Since the 180 is the size of zipfile-fixture.zip, would it make sense to do a `let origSize = fs.statSync(`${t.context.fixturedir}/zipfile-fixture.zip` (or something)? While it does add another line, it would make the test a bit more resilient to changes to the zip file.

Reply:
@ifestus  that's .... probably preferable for a fixture like this, as the contents are largely irrelevant, and it eliminates the magic-numbery feeling.     👍

Reply:
👍